.circleci/chrome_webstore_deploy.sh

```diff
@@ -68,7 +68,7 @@ else
   fi
   if [ "${CIRCLE_BRANCH}" == "staging" ]
```

Comment:
I think we can delete this one entirely, right? Along with the staging branch on Github too

Reply:
@rcmorano friendly ping

Reply:
I think that having a `staging` branch/environment should be a differentiated from using the `staging` cardano network, so if we still want to use the staging branch to have frozen/stable versions to test before releasing (which I think is a good practice, since `develop` is continuously hit with new merges all the time), we should keep that conditional and the staging branch to control what network we want to test the pre-release against (maybe in the future we need to test `staging` branch against `mainnet` for some reason).